### ISSUE TITLE
Create an initialize script.

### ISSUE THREAD

**Issue by razonyang** [razonyang/hugo-theme-bootstrap-skeleton#4]

**Comment by Rubonnek** [comment]
I think the initialization script should also clean up the baseURL override on the Github Pages workflow:

https://github.com/razonyang/hugo-theme-bootstrap-skeleton/blob/2cd9d748b4cc9ea093e51a533dd2d1f1b2487b78/.github/workflows/gh-pages.yml#L42

**Comment by razonyang** [comment]
@Rubonnek Yep, but still thinking which language should be used to write this script.
